modelforge/utils/prop.py

```diff
@@ -18,6 +19,16 @@ class PropertyNames:
     dipole_moment: Optional[str] = None
 
 
+PropertyUnits = {
+    "atomic_numbers": "dimensionless",
+    "positions": unit.nanometer,
+    "E": unit.kilojoule_per_mole,
+    "F": unit.kilojoule_per_mole / unit.nanometer,
+    "total_charge": unit.elementary_charge,
+    "dipole_moment": unit.elementary_charge * unit.nanometer,
+}
+
```

Comment:
that's really useful!

Reply:
YES, I think so too!  It should help make things clearer for anyone looking at the code.